src/styles/_catalog.scss

```diff
@@ -1,6 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(4, minmax(200px, 1fr));
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, minmax(200px, 1fr));
```

Comment:
I'm not sure that you have to use **minmax** here wth 1fr.. that makes big gaps between cards with changing screen's size:

![image](https://user-images.githubusercontent.com/106536680/174486442-fc76a1af-a25b-439e-9daf-d20cfbc242a3.png)

It is  better to use fixed cell's width: 200px